### PR TITLE
python3Packages.sphinxcontrib-excel-table: init at 1.0.8

### PR DESCRIPTION
**Pull request by raboof**
###### Motivation for this change

To build the docs at
https://github.com/collectiveaccess/ProvidenceDocs

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+buildPythonPackage rec {
+  pname = "sphinxcontrib-excel-table";
+  version = "1.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256:1q79byn3k3ribvwqafbpixwabjhymk46ns8ym0hxcn8vhf5nljzd";
+  };
+
+  buildInputs = [ nose ];
```
Comment (fabaff) [inline pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix:18]: `nose` is a `checkInputs`. `nose` should be removed as there are no tests (`Ran 0 tests in 0.000s`).

Please add `doCheck = false;` and `pythonImportsCheck`.

File: pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  propagatedBuildInputs = [ sphinx openpyxl ];
```
Comment (fabaff) [inline pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix:20]: There are more [requirements](https://github.com/hackerain/sphinxcontrib-excel-table/blob/master/setup.py#L40-L44). Are those not needed?

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125718` at 2c76e9e8 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.sphinxcontrib-excel-table</li>
    <li>python39Packages.sphinxcontrib-excel-table</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix:25:0:
```
   |
25 |     description = "Sphinx excel-table extension";
   | ^
```
</li>
  </ul>
</details>

**Review by fabaff — COMMENTED** [review]
Please update the commit messages (see https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes).

**Comment by raboof** [comment]
> Please update the commit messages (see https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes).

What's wrong with it? Should I prefix with `python3Packages.`?

File: pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix (hunk 2 above)
Comment (raboof) [reply pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix:20]: looks like it works without them - I suspect they come in transitively with `sphinx`. Would you prefer me to add them?

File: pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix (hunk 1 above)
Comment (raboof) [reply pkgs/development/python-modules/sphinxcontrib-excel-table/default.nix]: Done (`pythonImportsCheck` was already there)

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 125718` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.sphinxcontrib-excel-table</li>
<li>python39Packages.sphinxcontrib-excel-table</li>
</ul>
</details>

**Pull request merged**